### ISSUE TITLE
Currency updating should immedietly update currency_exchange_last_refresh and currency_exchange_historical_last_refresh key

### ISSUE THREAD

**Issue by applike-ss** [justtrackio/gosoline#620]
The currency updater module should immedietly update the currency_exchange_last_refresh and currency_exchange_historical_last_refresh key to ensure that multiple running tasks that implement the currency module don't start updating all nearly at the same time just because the value of the key is outdated.

**Comment by ajscholl** [comment]
Then we should also make sure we do some kind of "update this only if it still has the old value", otherwise you can still have two tasks updating the currency values at the same time if they both read and then write in the correct order.

**Comment by applike-ss** [comment]
yes, i was in for a lock as well. @j4k4 what do you think? To lock or not to lock, that is the question. :lock: :closed_lock_with_key:
